apps/frontend/src/app/5_pages/AavePage/components/BorrowAssetsList/BorrowAssetsList.tsx

```diff
@@ -68,6 +102,16 @@ export const BorrowAssetsList: FC = () => {
         orderOptions={orderOptions}
         setOrderOptions={setOrderOptions}
       />
+
+      <Dialog disableFocusTrap isOpen={!!borrowAssetDialog}>
```

Comment:
You 'd abstract this to a component like (not tested just an example): 

`
function BorrowAssetDialog({ isOpen, onClose, asset }) {
  const { t } = useTranslation();

  return (
    <Dialog disableFocusTrap isOpen={isOpen}>
      <DialogHeader
        title={t(translations.aavePage.common.borrow)}
        onClose={onClose}
      />
      <DialogBody className="flex flex-col gap-6">
        <BorrowForm asset={asset} onComplete={onClose} />
      </DialogBody>
    </Dialog>
  );
}`